dynamo-impl/src/main/java/org/dynamoframework/configuration/DynamoConfigurationProperties.java

```diff
@@ -0,0 +1,295 @@
+         */
+        private AttributeBooleanFieldMode booleanFieldMode = AttributeBooleanFieldMode.CHECKBOX;
+
+        /**
+         * The default date format
+         */
+        private String dateFormat = "dd-MM-yyyy";
+
+        /**
+         * The default date/time (time stamp) format
+         */
+        private String dateTimeFormat = "dd-MM-yyyy HH:mm:ss";
+        /**
+         * The default decimal precision
+         */
+        private Integer decimalPrecision = 2;
+
+        /**
+         * Indicates the default mode to use for element collection fields
+         */
+        private ElementCollectionMode elementCollectionMode = ElementCollectionMode.CHIPS;
+
+        /**
+         * The default field type to use for enumeration attributes
+         */
+        private AttributeEnumFieldMode enumFieldMode = AttributeEnumFieldMode.DROPDOWN;
+
+        /**
+         * The representation of the value <code>false</code>
+         */
+        private String falseRepresentation = "false";
+
+        /**
+         * Localized representations of the value <code>false</code>
+         */
+        private Map<String, String> falseRepresentations = new HashMap<>();
+
+        /**
+         * Localized representations of the value <code>true</code>
+         */
+        private Map<String, String> trueRepresentations = new HashMap<>();
+
+        /**
+         * The representation of the value <code>true</code>
+         */
+        private String trueRepresentation = "true";
+
+        /**
+         * The default group together mode
+         */
+        private GroupTogetherMode groupTogetherMode;
+
+        /**
+         * The column width from grouping together
+         */
+        private Integer groupTogetherWidth = 300;
+
+        /**
+         * The default locale
+         */
+        private Locale locale = Locale.ENGLISH;;
+
+        /**
+         * The default nesting depth
+         */
+        private Integer nestingDepth = 2;
+
+        /**
+         * The default number field mode
+         */
+        private NumberFieldMode numberFieldMode = NumberFieldMode.TEXTFIELD;
+
+        /**
+         * The default case sensitiveness for search
+         */
+        private boolean searchCaseSensitive = false;
+
+        /**
+         * Whether search is prefix only
+         */
+        private boolean searchPrefixOnly = false;
+
+        /**
+         * The default time format
+         */
+        private String timeFormat = "HH:mm:ss";
+
+        /**
+         * Indicates whether to use the display name as the input prompt by default
+         */
+        private boolean usePromptValue = true;
+
+        /**
+         * Whether to trim white space for text inputs
+         */
+        private boolean trimSpaces = false;
+
+        /**
+         * The default AI service
+         */
+        private String aiService;
+
+    }
+
+//    private EntityModelProperties entityModel = new EntityModelProperties();
```

Comment:
Niet meer nodig?

Reply:
Dit is niet meer nodig, de packages worden in ClassUtils nu dynamisch opgezocht